sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -760,16 +759,15 @@
         ],
         "responses": {
           "200": {
-            "description": "Returns the device twin object",
+            "description": "Returns the device twin object.",
```

Comment:
Is it not enough to say "Returns the device twin"? Do we need to add object?

Reply:
Sure, it is an object, so stating it again is redundant.